test/test_video_gpu_decoder.py

```diff
@@ -36,6 +36,33 @@ def test_frame_reading(self):
                     mean_delta = torch.mean(torch.abs(av_frames.float() - vision_frames.cpu().float()))
                     assert mean_delta < 0.75
 
+    @pytest.mark.skipif(av is None, reason="PyAV unavailable")
+    @pytest.mark.parametrize("keyframes", [True, False])
+    @pytest.mark.parametrize(
+        "full_path, duration",
+        [
+            (os.path.join(VIDEO_DIR, x), y)
+            for x, y in [
+                ("v_SoccerJuggling_g23_c01.avi", 8.0),
+                ("v_SoccerJuggling_g24_c01.avi", 8.0),
+                ("R6llTwEh07w.mp4", 10.0),
+                ("SOX5yA1l24A.mp4", 11.0),
+                ("WUzgd7C1pWA.mp4", 11.0),
+            ]
+        ],
+    )
+    def test_seek_reading(self, keyframes, full_path, duration):
+        decoder = VideoReader(full_path, device="cuda:0")
+        time = duration / 2
+        decoder.seek(time, keyframes_only=keyframes)
+        with av.open(full_path) as container:
+            container.seek(int(time * 1000000), any_frame=not keyframes, backward=False)
+            for av_frame in container.decode(container.streams.video[0]):
+                av_frames = torch.tensor(av_frame.to_rgb(src_colorspace="ITU709").to_ndarray())
+                vision_frames = next(decoder)["data"]
+                mean_delta = torch.mean(torch.abs(av_frames.float() - vision_frames.cpu().float()))
+                assert mean_delta < 0.75
```

Comment:
Why 0.75 ?

Reply:
There are some differences in pixel values that we get compared to pyav(not entirely sure what's the reason; could be rounding related),  I found that the mean of the differences is under 0.75 for the test videos. See https://github.com/pytorch/vision/blob/44ae1e51341faed0873c0b83096a6e64b72080ba/test/test_video_gpu_decoder.py#L37